### DISCUSSION TITLE
Exporting embedded notes, opening in Logseq

### DISCUSSION THREAD

**Discussion by anwxwna** [windingwind/zotero-better-notes#577] [Q&A]
I've been using this plugin for a day, it looks very promising, to be able to read and annotate in the same program. Thank you for making this plugin!

I am currently using linked attachments for all the attached pdfs, and I sync the attachment folder via OneNote. I read online that zotero stores its notes in the sqlite database. The picture snapshots are stored in the folder zotero creates in the storages folder. 

Now I see that BN is able to export a automatic sync markdown file, that can be opened by Logseq.  But I cannot open the linked notes from the main exported note, from Logseq or Obsidian. They seem to be zotero links, but do not open the corresponding file in zotero. 

Also when I export the main note as a markdown note with embedded pictures and notes (from the annotations I make for the paper), and then open it in Logseq, the images are not rendered. However it seems to render properly in Obsidian, when the same. 

Interestingly, if I export the note from the folder view (in zotero), by right-clicking and clicking export note, I am able to open links from logseq (it opens the notes workspace in zotero). This file however is not synced (despite also being listed in the Sync Manager)

Is there some setting I can change anywhere that can make the images render atleast in Logseq, or be able to open the linked notes links from outside zotero (Using logseq/obsidian)? I would prefer to use Logseq. 

Or is there a way to link to the knowledge graph itself, such that I can open it from Logseq?

**Comment by selfpowerful** [comment]
To render local images in logseq, you should use `![](file:///image_path)`. logseq doesn't have a stable version at the moment, so other problems could be due to logseq?
